bndtools.jareditor/src/bndtools/jareditor/internal/TemporaryProject.java

```diff
@@ -0,0 +1,133 @@
+package bndtools.jareditor.internal;
+
+import org.eclipse.core.resources.IContainer;
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IProjectDescription;
+import org.eclipse.core.resources.IWorkspace;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.SubMonitor;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.launching.JavaRuntime;
+
+public class TemporaryProject {
+
+	public static final String PROJECT_NAME = "JAREditorTempFiles";
```

Comment:
Perhaps `Bndtools` should be in the name so people can associate seeing this with Bndtools?

Reply:
Agreed, that would let people know where it is coming from when they encounter it.